tox.ini

```diff
@@ -10,6 +10,7 @@ docker_compose_version = 1.26.2
 # running outside of Yelp.
 indexserver = https://pypi.yelpcorp.com/simple
 basepython = python3.7
+usedevelop = true
```

Comment:
not necessary for this change, but this makes development a lot easier and i'm surprised we didn't already have this set

Reply:
This was causing issues in `k8s_itests` failing so I have removed this